### ISSUE TITLE
SHA1 Mismatch

### ISSUE THREAD

**Issue by qpre** [vercel/terraform-provider-vercel#53]
Hello,

I'm trying to use the provider to deploy an app within a monorepo (turborepo).

Whenever I try to deploy I get into SHA1 mismatches and I can't figure how nor why as there are no reason that the files taken from `data.vercel_project_directory` would change during the terraform run:

```
data "vercel_project_directory" "project" {
  path = "${path.root}/../../../../app/browser"
}

resource "vercel_project" "project" {
  name      = "awesome"
  framework = "nextjs"
  serverless_function_region = "cdg1"

  environment = []

  # https://vercel.com/docs/concepts/monorepos/turborepo
  build_command = "cd ../.. && npx turbo run build --filter=browser"
  root_directory = "app/browser"
}

resource "vercel_deployment" "project" {
  project_id    = vercel_project.project.id
  files              = data.vercel_project_directory.project.files
  path_prefix = 
  production  = true
}
```

And then I get:

```
Error: Error uploading deployment file
[1266](?check_suite_focus=true#step:13:1267)
│ 
[1267](?check_suite_focus=true#step:13:1268)
│   with module.platform.module.dashboard.vercel_deployment.project,
[1268](?check_suite_focus=true#step:13:1269)
│   on ../../recipes/vercel/main.tf line 16, in resource "vercel_deployment" "project":
[1269](?check_suite_focus=true#step:13:1270)
│   16: resource "vercel_deployment" "project" {
[1270](?check_suite_focus=true#step:13:1271)
│ 
[1271](?check_suite_focus=true#step:13:1272)
│ Could not upload deployment file
[1272](?check_suite_focus=true#step:13:1273)
│ ../../../../app/browser/packages/assets/images/bluetooth.svg, unexpected
[1273](?check_suite_focus=true#step:13:1274)
│ error: sha1sum_mismatch - SHA1 sum of the file
[1274](?check_suite_focus=true#step:13:1275)
│ (9bbe672f3b95401c035390bae38639915358c4f9) doesn't match the provided SHA1
[1275](?check_suite_focus=true#step:13:1276)
│ sum (296a875c1282c3f01a361f5f075faf99657d17e4)
```

I'm using version 0.6.2 of the provider and terraform 1.2.6, and it runs on GitHub actions.

**Comment by dglsparsons** [comment]
Hi @qpre, thanks for raising this. Sorry for the slow reply - I've been away on honeymoon. 
Could you share the operating system you are using? And if possible, could you share the `bluetooth.svg` file, in case it's something specific to that file?

**Comment by jimmybrancaccio** [comment]
I'm actually having a very similar issue. Here's the [failed GitHub Actions](https://github.com/jimmybrancaccio/learn-terraform-preview-environment/runs/7905159687?check_suite_focus=true) run. And it seems to be failing on [this file](https://github.com/jimmybrancaccio/learn-terraform-preview-environment/blob/main/public/favicon-152.png).

I am using macOS 10.15.x locally but this is a forked repo so I never actually uploaded that PNG file from my laptop.

**Comment by dglsparsons** [comment]
The file definitely has a windows line-ending it in. I'll have a dig and get this fixed asap. Thanks for the pointers @jimmybrancaccio !

**Comment by dglsparsons** [comment]
Hey @jimmybrancaccio, @qpre. This should be fixed as of v0.7.0. Please give it a go and let me know if the issue remains!
